rotkehlchen/accounting/constants.py

```diff
@@ -1,11 +1,19 @@
 from rotkehlchen.accounting.mixins.event import AccountingEventType
-from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
-from rotkehlchen.chain.evm.decoding.types import EventCategory, EventCategoryDetails
+from rotkehlchen.accounting.structures.types import (
+    EventCategory,
+    EventCategoryDetails,
+    EventDirection,
+    HistoryEventSubType,
+    HistoryEventType,
+)
 
 FREE_PNL_EVENTS_LIMIT = 1000
 FREE_REPORTS_LOOKUP_LIMIT = 20
 
-
+"""
+Default mapping for events added manually or from exchanges.
+Used as a fallback when the combination is not found in the standard mapping.
+"""
 DEFAULT_EVENT_CATEGORY_MAPPINGS = {
```

Comment:
As we mentioned this should go away since we are relaying on only a defined set of valid combinations